librz/arch/isa/msp430/msp430_disas.c

```diff
@@ -370,10 +431,14 @@ static int decode_oneop_opcode(ut16 instr, ut16 op, struct msp430_cmd *cmd) {
 			switch (get_dst(instr)) {
 			case MSP430_R3:
 				snprintf(cmd->operands, sizeof(cmd->operands), "#0");
+				cmd->dst = 0;
+				cmd->dst_mode = MSP430_IMM;
 				break;
 			default:
 				snprintf(cmd->operands, sizeof(cmd->operands),
 					"%s", msp430_register_names[get_dst(instr)]);
+				cmd->dst = get_dst(instr);
+				cmd->dst_mode = MSP430_REG;
```

Comment:
```suggestion
				cmd->dst = get_dst(instr);
				cmd->dst_mode = MSP430_REG;
				break;
```

Reply:
Also below.

Reply:
There is an assignment below it that sets the return value, so it shouldn't break.

Reply:
This gets set anyways, the `break;` only exits the `switch`. But it is optional anyways. I just like to have any `case/default` with a `break`. Maybe someone changes the order of the `default` and other `cases` in the future and forgets it the `break`. This was my thinking here